config/environments/staging.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+Swapsea2::Application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
+
+  # Code is not reloaded between requests.
+  config.cache_classes = true
+
+  # Eager load code on boot. This eager loads most of Rails and
+  # your application in memory, allowing both thread web servers
+  # and those relying on copy on write to perform better.
+  # Rake tasks automatically ignore this option for performance.
+  config.eager_load = true
+
+  # Full error reports are disabled and caching is turned on.
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
+
+  # Enable Rack::Cache to put a simple HTTP cache in front of your application
+  # Add `rack-cache` to your Gemfile before enabling this.
+  # For large-scale production use, consider using a caching reverse proxy like nginx, varnish or squid.
+  # config.action_dispatch.rack_cache = true
+
+  # Disable Rails's static asset server (Apache or nginx will already do this).
+  config.serve_static_assets = false
+
+  # Compress JavaScripts and CSS.
+  config.assets.js_compressor = :uglifier
+  # config.assets.css_compressor = :sass
+
+  # Do not fallback to assets pipeline if a precompiled asset is missed.
+  config.assets.compile = false
+
+  # Add the fonts path
+  config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
+
+  # Precompile additional assets
+  config.assets.precompile += %w[
+    .svg .eot .woff .ttf
+
+    admin-layout.css admin-layout.js
```

Comment:
Does `precompile` compile `.css` and `.js` by default?

Reply:
This probably best explains it. We could certainly refactor this though.

https://guides.rubyonrails.org/asset_pipeline.html#precompiling-assets



<img width="706" alt="Screen Shot 2021-08-13 at 5 42 52 pm" src="https://user-images.githubusercontent.com/4843657/129322309-3d461fca-627b-4f81-a158-72c0043d1734.png">